src/Workleap.OpenApi.MSBuild/OasdiffManager.cs

```diff
@@ -42,14 +40,13 @@ public async Task RunOasdiffAsync(IEnumerable<string> openApiSpecFiles, IEnumera
         var generatedOpenApiSpecFilesList = generatedOpenApiSpecFiles.ToList();
         var oasdiffExecutePath = Path.Combine(this._oasdiffDirectory, RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "oasdiff.exe" : "oasdiff");
 
-        foreach (var specFile in openApiSpecFiles)
+        foreach (var baseSpecFile in openApiSpecFiles)
         {
-            this._loggerWrapper.LogMessage($"Starting Oasdiff comparison with {specFile}.");
+            this._loggerWrapper.LogMessage($"Starting Oasdiff comparison with {baseSpecFile}.");
             
-            var fileName = Path.GetFileName(specFile);
-            var baseSpecRelativePath = Path.GetRelativePath(this._openApiToolsDirectory, specFile);
-            var newSpecRelativePath = Path.GetRelativePath(this._openApiToolsDirectory, generatedOpenApiSpecFilesList.First(x => x.Contains(fileName)));
-            await this._processWrapper.RunProcessAsync(oasdiffExecutePath, new[] { "diff", baseSpecRelativePath, newSpecRelativePath, "--exclude-elements", "description,examples,title,summary", "-f", "text" }, cancellationToken);
+            var fileName = Path.GetFileName(baseSpecFile);
+            var newSpecRelativePath = generatedOpenApiSpecFilesList.First(x => x.Contains(fileName));
```

Comment:
Is it possible that this one could be null? If it's possible, should we error out?